### ISSUE TITLE
The return value of FlutterTexture:copyPixelBuffer should be nullable

### ISSUE THREAD

**Issue by tvolkert** [flutter/flutter#16165]
(from [FlutterTexture.h](https://github.com/flutter/engine/blob/04f3c6557c02879191a558320ed8ec81f8b2b619/shell/platform/darwin/ios/framework/Headers/FlutterTexture.h#L17)):

```objc
@protocol FlutterTexture<NSObject>
- (CVPixelBufferRef)copyPixelBuffer;
@end
```

`CVPixelBufferRef` should be set to nullable. It becomes problematic when we use this API in swift where nil value is allowed for `CVPixelBufferRef`. 

Note: the above was transferred from an internal bug.

**Comment by cbracken** [comment]
Fixed by flutter/engine#4934

**Comment by cbracken** [comment]
Dupe of #16041
